### ISSUE TITLE
Choose color for current item in Tree struture

### ISSUE THREAD

**Issue by eueiwojfudn** [giuspen/cherrytree#1456]
Hi,

I like to work in black for text and white for background. This is my choice for the menu, editor and tree structure.

I've noticed when I select text in the editor or when I take a look at submenu, the current item is "blue as background and white as text" which is OK. The contrast if very good.

But when I select an item in the Tree structure, the text remains black and the background goes to blue. 

The ideal, in my point of view, would be that the tree structure has the same behavior and contrast for the current item as in the editor and menu.

In other words, in my example, "white for text" and "blue for background" for the current item in the tree structure. An excellent contrast.

Maybe there is an option to change the color of the text (current) in the tree structure, but I didn't find it.

By the way, this is a detail for accessibility. No problem if it remains the way it is.

Thank you for excellent work.

**Comment by giuspen** [comment]
I made the selected text color of the tree consistent with that in the menu, will be in 0.99.28
